cmd/swarmctl/service/update.go

```diff
@@ -36,6 +36,16 @@ var (
 				return err
 			}
 
+			// change the image if that positional arg is present
+			if len(args) > 1 {
+				spec.Task.GetContainer().Image = args[1]
+			}
+
+			// change the args too, if those positional args are present
```

Comment:
Feels weird that I have to specify the image on the command line if I want to update the command/arguments. Looks like docker uses `--image` for update; I guess we should be consistent with that.

But `docker service update` looks like it doesn't handle args gracefully (the very thing we're trying to fix here). I'll file a bug against it.